data/js/item/bagofsending.js

```diff
@@ -0,0 +1,177 @@
+function onCreateDFN( objMade, objType )
+{
+	const maxCharges = 30;
+	if( objType == 0 ) 
+	{
+		var Charges = 0;
+		switch( RandomNumber( 0, 2 ))
+		{
+			case 0: Charges = 3; break;
+			case 1: Charges = 6; break;
+			case 2: Charges = 9; break;
+		}
+		objMade.SetTag( "charges", Charges )
```

Comment:
Doesn't this mean the bag of sending will always have 3/30 chargers, 6/30 charges or 9/30 charges, since maxCharges is defined (and set) as 30, while the number of charges is always set as 3, 6 or 9?

Also, rename "Charges" variable to "charges"? ;)